galaxy_ng/app/api/ui/viewsets/distribution.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework import mixins
+from pulp_ansible.app import models as pulp_models
+from guardian.shortcuts import get_objects_for_user
+
+from galaxy_ng.app.access_control import access_policy
+from galaxy_ng.app.api.ui import serializers, versioning
+from galaxy_ng.app.api import base as api_base
+from galaxy_ng.app import models
+
+
+class DistributionViewSet(
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    api_base.GenericViewSet,
+):
+    serializer_class = serializers.DistributionSerializer
+    model = pulp_models.AnsibleDistribution
+    queryset = pulp_models.AnsibleDistribution.objects.exclude(name__startswith='inbound-') \
+        .exclude(name__endswith='-synclist')
```

Comment:
Just a suggestion, I prefer to avoid the  `\`  when  parenthesis can be used to multi line.
```suggestion
    queryset = pulp_models.AnsibleDistribution.objects.exclude(
        name__startswith='inbound-'
    ).exclude(name__endswith='-synclist')
```